Update/omot_012.txt

```diff
@@ -1991,10 +1991,10 @@ void main()
 			NULL, "\"...Zzz~...\"", Line_Normal);
 	ClearMessage();
 
-//rとりあえず落ち着いたのか、穏やかな寝息を立てている魅音。…鼻血の後がちょっとコミカルだが、一応女の子なので見なかったことにしてやろう。
+//rとりあえず落ち着いたのか、穏やかな寝息を立てている魅音。…鼻血の跡がちょっとコミカルだが、一応女の子なので見なかったことにしてやろう。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
-	OutputLine(NULL, "とりあえず落ち着いたのか、穏やかな寝息を立てている魅音。…鼻血の後がちょっとコミカルだが、一応女の子なので見なかったことにしてやろう。",
-			NULL, "At least for now, things seemed to have calmed down with Mion peacefully asleep. ...It was kind of amusing to see her with a nosebleed, but since she's a girl, I'll pretend I never saw it.", Line_Normal);
+	OutputLine(NULL, "とりあえず落ち着いたのか、穏やかな寝息を立てている魅音。…鼻血の跡がちょっとコミカルだが、一応女の子なので見なかったことにしてやろう。",
+			NULL, "At least for now, things seemed to have calmed down with Mion peacefully asleep. ...It was kind of amusing to see her with a stain of nosebleed, but since she's a girl, I'll pretend I never saw it.", Line_Normal);
```

Comment:
```suggestion
			NULL, "At least for now, things seemed to have calmed down with Mion peacefully asleep. ...It was kind of amusing to see her with a nosebleed stain, but since she's a girl, I'll pretend I never saw it.", Line_Normal);
```